assets/_scss/components/_alerts.scss

```diff
@@ -9,11 +6,13 @@
   margin-top: 1.5em;
   padding: 1em;
 
+  @include media($medium-screen) {
+    background-size: 5.2rem;
```

Comment:
This `rem` unit is for the icon image, where as the rest applies for text. I don't think there's a need to have a huge icon image if there's larger text. Correct, in this case the margin top would change depending on the parent text size.